README.md

```diff
@@ -1115,33 +1115,39 @@
 
 ### Linux News, Apps, and more:
 
-* [OMG!Ubuntu](https://www.omgubuntu.co.uk/)
-* [ITSFOSS](https://itsfoss.com/)
-* [Linux official](https://www.linux.com/)
-* [Webupd8](http://www.webupd8.org/)
-* [Noobslab](https://www.noobslab.com/)
-* [Make use of](https://www.makeuseof.com/service/linux/)
-* [TecMint](https://www.tecmint.com/)
+* [9To5Linux](https://9to5linux.com/)
 * [AllTop](https://linux.alltop.com/)
-* [Unixmen](https://www.unixmen.com/)
-* [DistroWatch](https://distrowatch.com/)
-* [Phoronix](https://www.phoronix.com/)
+* [ArchiveOS](https://archiveos.org/linux/)
 * [Dedoimedo](https://www.dedoimedo.com/)
+* [DistroTube](https://distrotube.com/)
+* [DistroWatch](https://distrowatch.com/)
+* [FAMILUG](https://www.familug.org/)
+* [GamingOnLinux](https://www.gamingonlinux.com/)
 * [How-To Geek](https://www.howtogeek.com/t/linux/)
+* [ITSFOSS](https://itsfoss.com/)
+* [Lemmy c/Linux](https://lemmy.ml/c/linux)
 * [Liliputing](https://liliputing.com/)
-* [FAMILUG](https://www.familug.org/)
-* [Ubuntu Geek](https://www.ubuntugeek.com/)
-* [LinuxStans](https://linuxstans.com/)
-* [9To5Linux](https://9to5linux.com/)
-* [UbuntuHandbook](https://ubuntuhandbook.org/)
 * [LinuxHandbook](https://linuxhandbook.com/)
-* [Open Source](https://opensource.com/)
-* [Slashdot](https://linux.slashdot.org/)
-* [LWN](https://lwn.net/)
-* [GamingOnLinux](https://www.gamingonlinux.com/)
-* [DistroTube](https://distrotube.com/)
 * [LinuxLinks](https://www.linuxlinks.com/)
+* [Linux official](https://www.linux.com/)
 * [LinuxSmartphones](https://linuxsmartphones.com/)
+* [LinuxStans](https://linuxstans.com/)
+* [Linux Timeline](https://github.com/FabioLolix/LinuxTimeline/releases)
+* [LWN](https://lwn.net/)
+* [Make use of](https://www.makeuseof.com/service/linux/)
+* [Nixcraft forum](https://www.nixcraft.com/)
+* [Nixcraft](https://www.cyberciti.biz/)
+* [Noobslab](https://www.noobslab.com/)
+* [OMG!Ubuntu](https://www.omgubuntu.co.uk/)
+* [Open Source](https://opensource.com/)
+* [Phoronix](https://www.phoronix.com/)
+* [Repology](https://repology.org/)
```

Comment:
I am not sure we should add this page. To me, it s not a news page but plainly a place to track package information. Am I mistaken by any means? Or do you have any ideas with this page that I haven't thought of ?